modules/core/src/core/transform.js

```diff
@@ -221,4 +304,118 @@ export default class Transform {
       });
     }
   }
+
+  // create buffer to access source texture's individual pixels.
+  _createElementIDBuffer(elementCount) {
+    if (!this.hasSourceTextures) {
+      return;
+    }
+    if (this.elementIDBuffer) {
```

Comment:
Nit: use `buffer.setData` to reinitialize an existing buffer. Should be faster than destroying and creating a new buffer.